CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.35.0
+
+- Propose-downstream: log when a PR already exists downstream (#1322).
+- `packit init` to set spec file path in the config if it's not defined (#1313).
+- Don't use `str.removeprefix()` since it was introduced in Python 3.9 and we still support older versions of Python 3 (#1318).
+- Make it possible to clone packages from staging dist-git (#1306).
+- Source-git: squash patches by patch name - no need to have a dedicated attribute, `squash_commits`, for that (#1309).
+- Source-git: look for the config file in .distro/source-git.yaml as well (#1302).
+- Source-git: don't raise an exception when a patch with final name already exists (#1316).
```

Comment:
that just fixes a bug introduced 2 days before the commit

Reply:
okay, didn't know that